### PR TITLE
limit number of attacks on sites with 200 codes

### PR DESCRIPTION
**Pull request by JJerboa**

### CODE REVIEW

**Comment by mlushpenko** [comment]
Seems to be broken after this change:
```
 File "/nowarddos/attack.py", line 102
    logger.info(f"ATTACKED {site}; attack count: {attacks_number}; RESPONSE CODE: {response.status_code}")
IndentationError: unexpected indent
```

File: attack.py [hunk 1]
```diff
@@ -87,19 +87,19 @@ def mainth(site: str):
...
+                        logger.info(f"ATTACKED {site}; attack count: {attacks_number}; RESPONSE CODE: {response.status_code}")
         else:
-            while True:
+            while (attacks_number < settings.MAX_REQUESTS_TO_SITE):
                 response = scraper.get(site, timeout=10)
                 if response.status_code >= 400:
                     break
                 attacks_number += 1
-                logger.info(f"ATTACKED {site}; RESPONSE CODE: {response.status_code}")
+                        logger.info(f"ATTACKED {site}; attack count: {attacks_number}; RESPONSE CODE: {response.status_code}")
```
Comment (mlushpenko) [inline attack.py:102]: Problem is here

Comment (mlushpenko) [reply attack.py:102]: @JJerboa

Comment (viachmev) [reply attack.py:102]: Confirm that there is an issue here:
```
 % python attack.py 1000 -v
  File "attack.py", line 102
    logger.info(f"ATTACKED {site}; attack count: {attacks_number}; RESPONSE CODE: {response.status_code}")
    ^
```

**Pull request merged**